app/policies/workspace_policy.rb

```diff
@@ -0,0 +1,7 @@
+class WorkspacePolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+end
```

Comment:
same as the other comments :) If it where the booking policy i would think that this is enough but maybe its engough for the current state anyways